save-common/src/commonMain/kotlin/org/cqfn/save/core/utils/ProcessBuilder.kt

```diff
@@ -149,6 +153,7 @@ class ProcessBuilder(private val useInternalRedirections: Boolean, private val f
         } else {
             command
         }
+        logTrace("Modifying cmd: $cdCmd")
```

Comment:
Please also add logging about what `pkill` command is executed in `ProcessBuilderInternal`